src/libraries/System.Text.RegularExpressions/src/dotnet-regular-expressions.md

```diff
@@ -0,0 +1,243 @@
+# Implementation of System.Text.RegularExpressions
```

Comment:
Nit: why did you choose dotnet-regular-expressions.md as the file name rather than, say, README.md?

Reply:
changed